extensions/amp-a4a/0.1/real-time-config-manager.js

```diff
@@ -50,14 +50,13 @@ export const RTC_ERROR_ENUM = {
 };
 
 /**
- * @param {!Array<!Promise<!rtcResponseDef>>} promiseArray
  * @param {string} error
  * @param {string} callout
  * @private
  */
-function logAndAddErrorResponse_(promiseArray, error, callout) {
+function logAndAddErrorResponse_(error, callout) {
```

Comment:
Instead of a new function, consider adding parameter to buildErrorResponse_ that causes logging (e.g. opt_log)

Reply:
done